### PR TITLE
Let the user configure the maximum points

### PR DESCRIPTION
**Pull request by haraldh**
Let the user configure the maximum points in Garmin connect App settings

### CODE REVIEW

**Comment by haraldh** [comment]
Oh, btw, you can't test the settings in the connect phone app, until the app is live in the store.

Learned it from:
https://forums.garmin.com/showthread.php?340820-WatchApp-add-many-DataField-into-a-single-view

**Comment by matco** [comment]
Thanks again for your contribution! I will try to merge and publish this tonight.

And yeah I am aware of the mess about app settings.

File: resources/properties.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<resources>
+    <properties>
+        <property id="maximum_points" type="number">21</property>
+        <property id="absolut_maximum_points" type="number">30</property>
```
Comment (matco) [inline resources/properties.xml]: There is a typo in "absolut".

You should stop drinking vodka :)

File: resources/settings.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<resources>
+    <settings>
+        <setting propertyKey="@Properties.maximum_points" title="@Strings.maximum_number_points_title">
+            <settingConfig type="numeric" />
+        </setting>
+
+        <setting propertyKey="@Properties.absolut_maximum_points" title="@Strings.absolute_maximum_number_points_title">
```
Comment (matco) [inline resources/settings.xml]: Same here, typo in "absolut".

**Comment by matco** [comment]
I just added some comments. Could you please indent with tabs? Thanks!

**Comment by haraldh** [comment]
force pushed an update with s/absolut_/absolute_/g and tabified

thanks for reviewing

**Pull request merged**